src/Xamarin.Android.Build.Tasks/Tasks/BuildApk.cs

```diff
@@ -43,6 +43,9 @@ public class BuildApk : Task
 		[Required]
 		public ITaskItem[] NativeLibraries { get; set; }
 
+		[Required]
+		public ITaskItem[] ApplicationDSOs { get; set; }
```

Comment:
Probably silly request, but could we please use "SharedLibraries" instead of "DSOs"?

Reply:
Sure :)